docker-compose.yml

```diff
@@ -3,8 +3,7 @@ services:
   main:
     build:
       context: ./docker
-      tags:
-        - automatisch/automatisch
+    image: automatisch/automatisch
```

Comment:
@ksurl could we use automatischio/automatisch as `automatischio` is the organisation name we got on docker hub and we plan to publish the container as `automatisch`?